src/pages/CookieClicker.vue

```diff
@@ -1,49 +1,103 @@
 <script setup>
-import { computed, ref } from 'vue';
+import { ref, computed, onMounted, onBeforeUnmount } from 'vue';
 
 const cookies = ref(0);
 const buildings = ref([
     { name: 'Cursor', price: 15, cps: 0.1, count: 0},
     { name: 'Grandma', price: 100, cps: 1, count: 0},
     { name: 'Farm', price: 1000, cps: 10, count: 0},
-    { name: 'Factory', price: 10_000, cps: 100, count: 0},
+    { name: 'Factory', price: 100000, cps: 100, count: 0},
```

Comment:
Feel free to leave _ in between as this is used to make it easier to understand big numbers. _ itself does nothing but you can more easier understand wether it is 10k or 100k